views/StudyDetails.react.js

```diff
@@ -0,0 +1,197 @@
+import React, { useState, useEffect } from 'react'
+import { compose } from 'redux';
+import { withStyles } from '@material-ui/core/styles';
+import { connect } from 'react-redux';
+import Typography from '@material-ui/core/Typography';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import Snackbar from '@material-ui/core/Snackbar';
+import Button from '@material-ui/core/Button';
+import Tooltip from '@material-ui/core/Tooltip';
+import Add from '@material-ui/icons/Add';
+import Delete from '@material-ui/icons/Delete';
+import Header from './components/Header';
```

Comment:
This is a nit but what do you think about separating node modules and local imports with a newline?
```suggestion
import Delete from '@material-ui/icons/Delete';

import Header from './components/Header';
```

Reply:
I can do that, what about for other imports like utils, should they be separated by a new line?

Reply:
I haven't done that in the past but if there is a convention in this app we can follow that.

Reply:
[f682564](https://github.com/AMP-SCZ/dpdash/pull/81/commits/f682564832e4de21cbe2ec9f3f4245bfeaef011a)